apps/game/src/layer/elevation.rs

```diff
@@ -144,7 +144,7 @@ impl SteepStreets {
             pt.project_away(arrow_len, Angle::degrees(135.0)),
         ])
         .make_polygons(thickness)
-        .scale(5.0);
+        .must_scale(5.0);
```

Comment:
Many callers of `scale` have a fixed input > 1, so no need to deal with a `Result`